src/assets/scripts/client/info/SimClockController.js

```diff
@@ -1,29 +1,51 @@
 import TimeKeeper from '../engine/TimeKeeper';
 import { digits_integer } from '../utilities/radioUtilities';
-import { SELECTORS } from '../constants/selectors';
 import { TIME } from '../constants/globalConstants';
 
 /**
  * Manages a clock that stays in sync with the current game time
- * @class GameClockView
+ * @class SimClockController
  */
-export default class GameClockView {
+export default class SimClockController {
     /**
-     * @for GameClockView
+     * @for SimClockController
      * @constructor
      */
-    constructor($element) {
-        this.$element = $element;
+    constructor() {
+        /**
+         * @for SimClockController
+         * @property startTime
+         * @type {Number}
+         * @default 0
+         */
         this.startTime = 0;
+
+        /**
+         * @for SimClockController
+         * @property time
+         * @type {Number}
+         * @default 0
+         */
         this.time = 0;
```

Comment:
how does `time` differ from `startTime` and how is it used?  this information should be in the docblock